src/java/main/org/apache/zookeeper/client/ZKClientConfig.java

```diff
@@ -56,9 +56,15 @@
     @SuppressWarnings("deprecation")
     public static final String SECURE_CLIENT = ZooKeeper.SECURE_CLIENT;
     public static final int CLIENT_MAX_PACKET_LENGTH_DEFAULT = 4096 * 1024; /* 4 MB */
+    public static final String ZOOKEEPER_REQUEST_TIMEOUT = "zookeeper.request.timeout";
+    /**
+     * Feature is disabled by default.
```

Comment:
What about enabling it by default with a default computed from desired session expire timeout?

Reply:
Session timeout and this client request timeout are completely different. I think we should not mix these two.

Reply:
Okay, no feafult